src/dune_threaded_console/dune_threaded_console.ml

```diff
@@ -61,7 +61,7 @@ let make (module Base : S) : (module Dune_console.Backend) =
     let start () =
       Base.start ();
       Dune_engine.Scheduler.spawn_thread @@ fun () ->
-      ignore (Unix.sigprocmask SIG_UNBLOCK [ Signal.to_int Winch ] : int list);
+      if Sys.unix then ignore (Unix.sigprocmask SIG_UNBLOCK [ Signal.to_int Winch ] : int list);
```

Comment:
So cygwin doesn't emulate signals on windows?

Reply:
I don't know, I never use Cygwin executables, but I can change the condition to `not Sys.win32` if that sounds better.

Reply:
I did the change.